src/picterra/tracer_client.py

```diff
@@ -288,3 +287,131 @@ def list_plots_analyses(
         if page_number is not None:
             data["page_number"] = int(page_number)
         return self._return_results_page(f"plots_groups/{plots_group_id}/analysis/", data)
+
+    def list_plots_analysis_reports(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str,
+    ):
+        """
+        List all the reports belonging to a given plots analysis
+
+        Args:
+            plots_group_id: id of the plots group on which we want to list the analyses
+            plots_analysis_id: id of the plots analysis for which we want to list the reports
+
+        Returns:
+            ResultsPage: A ResultsPage object that contains a slice of the list
+                of plots analyses reports dictionaries
+        """
+        return self._return_results_page(
+            f"plots_groups/{plots_group_id}/analysis/{plots_analysis_id}/reports/"
+        )
+
+    def list_plots_analyses_report_types(
+        self,
+        plots_group_id: str,
+        plots_analysis_id: str
```

Comment:
It's probably a follow up / nice to have, but for consistency I think having a `search` param like we do for `list_methodologies` would be nice

Reply:
I'm opening a follow-up in GL because we need to update the Django view to allow search